### PR TITLE
Added Java in Hindi Book

### PR DESCRIPTION
**Pull request by anishhh238**
Added Java in Hindi Book pdf link from where student can download pdf of JAVA lang.

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Added link of Java in Hindi book pdf

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by anishhh238** [comment]
look over this PR

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="java"></a>Java
```
Comment (shrn01) [inline books/free-programming-books-hi.md:14]: This is just a sample of the actual book. We do not accept samples

**Comment by shrn01** [comment]
Please provide the complete book if it is free or please close this PR

**Comment by eshellman** [comment]
sample, not the whole book

**Pull request closed without merging**